api/nakadi-event-bus-api.yaml

```diff
@@ -606,22 +488,394 @@ paths:
           description: Partition id
           type: string
           required: true
-        - name: X-Flow-Id
-          in: header
+        - $ref: '#/parameters/XFlowId'
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/Partition'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /subscriptions:
+    put:
+      tags:
+        - managed-api
```

Comment:
there's multiple terms for this api - "managed", "subscription", "high level" ("hila"). Is there a way to reduce the terminology needed?